tests/integration/security/util/framework.go

```diff
@@ -48,4 +48,10 @@ func EchoConfig(name string, ns namespace.Instance, headless bool, annos echo.An
 		Galley: g,
 		Pilot:  p,
 	}
+
+	// for headless service, the port and target port must be equal
+	if headless {
+		out.Ports[0].ServicePort = 8090
```

Comment:
hardcoding port causes some problem. is there any constants from echo config for this port?  @nmittler wdty?

Reply:
This is same as above L37, if no change `InstancePort` there, this is not neccessary.